packages/cmr-client/tests/test-searchConcept.js

```diff
@@ -4,20 +4,22 @@ const got = require('got');
 const sinon = require('sinon');
 const test = require('ava');
 
-const searchConcept = require('../searchConcept');
+const { searchConcept } = require('../searchConcept');
+
+process.env.CMR_ENVIRONMENT = 'SIT';
 
 const clientId = 'cumulus-test-client';
 
 test.serial('searchConcept request includes CMR client id', async (t) => {
   let request;
   const stub = sinon.stub(got, 'get').callsFake((_url, opt) => {
     request = { headers: opt.headers };
-    return { body: { feed: { entry: [] } }, headers: { 'cmr-hits': 0 } };
+    return { body: { feed: { entry: [] } }, headers: { 'cmr-hits': '0' } };
```

Comment:
I probably missed this in the code, but why this change?

Reply:
`URLSearchParams` only takes strings and null